src/game/BattleGround/BattleGroundWS.cpp

```diff
@@ -581,12 +585,12 @@ void BattleGroundWS::FillInitialWorldStates(WorldPacket& data, uint32& count)
 
     FillInitialWorldState(data, count, BG_WS_FLAG_CAPTURES_MAX, BG_WS_MAX_TEAM_SCORE);
 
-    if (m_FlagState[TEAM_INDEX_HORDE] == BG_WS_FLAG_STATE_ON_PLAYER)
+    if (m_FlagState[TEAM_INDEX_ALLIANCE] == BG_WS_FLAG_STATE_ON_PLAYER)
         FillInitialWorldState(data, count, BG_WS_FLAG_STATE_HORDE, 2);
     else
         FillInitialWorldState(data, count, BG_WS_FLAG_STATE_HORDE, 1);
-
-    if (m_FlagState[TEAM_INDEX_ALLIANCE] == BG_WS_FLAG_STATE_ON_PLAYER)
+    
```

Comment:
this, whatever this is

Reply:
oops sorry lol I'll correct

Reply:
ye it's a werid one if I catch your drift, it seems the world states are getting mixed up somehow. I don't really know the inner workings on how the data bits are being applied, just a quick hack fix to get it working for the most part. Need to test BenDol's rework out though as anything is better than my work arounds :D